server/app/services/settings/SettingsManifest.java

```diff
@@ -2049,6 +2054,13 @@ public boolean getNorthStarApplicantUi(RequestHeader request) {
                   + " enable or disable in-development features.",
               ImmutableList.of(),
               ImmutableList.of(
+                  SettingDescription.create(
+                      "ELIGIBILITY_MESSAGE_ENABLED",
+                      "Enables civiform admins to set up a customized eligibility message per"
```

Comment:
**suggestion:** Can you add the "(NOT FOR PRODUCTION USE)" caveat to this as well?

Reply:
yeah for sure! done

Reply:
I still don't see it?

Reply:
Oo I added the phrase at SettingsManifest, which got formatted to its original state since it didn't match description in env-var-docs. But now it should all be the same with the phrase "(NOT FOR PRODUCTION USE)"